packages/acceptance-tests/pkg-tests-specs/sources/node-modules.test.ts

```diff
@@ -169,6 +169,47 @@ describe(`Node_Modules`, () => {
     ),
   );
 
+  test(`should not treat link: dependencies as an inner workspaces`,
```

Comment:
I don't know how great this test name is - the `expect()` assertion at the end kind of seems unrelated to it, so I'm happy to take suggestions.

Reply:
I think the description is fine